Tests are having compile error due to doughnut

https://circleci.com/gh/cennznet/plug-blockchain/72?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link#


```
Running `rustc --edition=2018 --crate-name srml_executive srml/executive/src/lib.rs --color never --emit=dep-info,link -C debuginfo=2 --test --cfg 'feature="default"' --cfg 'feature="parity-codec"' --cfg 'feature="primitives"' --cfg 'feature="rstd"' --cfg 'feature="runtime_io"' --cfg 'feature="serde"' --cfg 'feature="srml-support"' --cfg 'feature="std"' --cfg 'feature="substrate-primitives"' --cfg 'feature="system"' -C metadata=a4b7407d998ac9a1 -C extra-filename=-a4b7407d998ac9a1 --out-dir /home/circleci/project/target/debug/deps -C incremental=/home/circleci/project/target/debug/incremental -L dependency=/home/circleci/project/target/debug/deps --extern hex_literal=/home/circleci/project/target/debug/deps/libhex_literal-ad53fd589ffd34fa.rlib --extern parity_codec=/home/circleci/project/target/debug/deps/libparity_codec-00f76678648a1a37.rlib --extern serde=/home/circleci/project/target/debug/deps/libserde-799fa49187662483.rlib --extern runtime_io=/home/circleci/project/target/debug/deps/libsr_io-bf478c53f86a882e.rlib --extern primitives=/home/circleci/project/target/debug/deps/libsr_primitives-11f183d831b88e81.rlib --extern rstd=/home/circleci/project/target/debug/deps/libsr_std-1259ba0ee2f0a2df.rlib --extern balances=/home/circleci/project/target/debug/deps/libsrml_balances-b0d013a0df1b3267.rlib --extern srml_indices=/home/circleci/project/target/debug/deps/libsrml_indices-fecf3245ac6fb095.rlib --extern srml_support=/home/circleci/project/target/debug/deps/libsrml_support-38946422098329d2.rlib --extern system=/home/circleci/project/target/debug/deps/libsrml_system-832b4575e9b3c7f0.rlib --extern substrate_primitives=/home/circleci/project/target/debug/deps/libsubstrate_primitives-66f45bfa70fe58f7.rlib -L native=/home/circleci/project/target/debug/build/clear_on_drop-e045ff54f593d332/out -L native=/home/circleci/project/target/debug/build/backtrace-sys-f1acc7cc7d33d508/out`
error[E0599]: no function or associated item named `initialize_block` found for type `doughnut::DoughnutExecutive<doughnut::tests::Runtime, sr_primitives::testing::Block<sr_primitives::testing::doughnut::TestXt<srml_balances::Call<doughnut::tests::Runtime>, sr_primitives::testing::doughnut::DummyDoughnut>>, srml_system::ChainContext<doughnut::tests::Runtime>, srml_balances::Module<doughnut::tests::Runtime>, ()>` in the current scope
   --> srml/executive/src/doughnut/mod.rs:289:24
    |
55  | / pub struct DoughnutExecutive<System, Block, Context, Payment, AllModules>(
56  | |     PhantomData<(System, Block, Context, Payment, AllModules)>,
57  | | );
    | |__- function or associated item `initialize_block` not found for this
...
289 |               Executive::initialize_block(&Header::new(
    |                          ^^^^^^^^^^^^^^^^ function or associated item not found in `doughnut::DoughnutExecutive<doughnut::tests::Runtime, sr_primitives::testing::Block<sr_primitives::testing::doughnut::TestXt<srml_balances::Call<doughnut::tests::Runtime>, sr_primitives::testing::doughnut::DummyDoughnut>>, srml_system::ChainContext<doughnut::tests::Runtime>, srml_balances::Module<doughnut::tests::Runtime>, ()>`
    |
    = note: the method `initialize_block` exists but the following trait bounds were not satisfied:
            `[u8; 64] : std::convert::AsRef<[u8]>`

error[E0599]: no function or associated item named `apply_extrinsic` found for type `doughnut::DoughnutExecutive<doughnut::tests::Runtime, sr_primitives::testing::Block<sr_primitives::testing::doughnut::TestXt<srml_balances::Call<doughnut::tests::Runtime>, sr_primitives::testing::doughnut::DummyDoughnut>>, srml_system::ChainContext<doughnut::tests::Runtime>, srml_balances::Module<doughnut::tests::Runtime>, ()>` in the current scope
   --> srml/executive/src/doughnut/mod.rs:296:24
    |
55  | / pub struct DoughnutExecutive<System, Block, Context, Payment, AllModules>(
56  | |     PhantomData<(System, Block, Context, Payment, AllModules)>,
57  | | );
    | |__- function or associated item `apply_extrinsic` not found for this
...
296 |               Executive::apply_extrinsic(xt).unwrap();
    |                          ^^^^^^^^^^^^^^^ function or associated item not found in `doughnut::DoughnutExecutive<doughnut::tests::Runtime, sr_primitives::testing::Block<sr_primitives::testing::doughnut::TestXt<srml_balances::Call<doughnut::tests::Runtime>, sr_primitives::testing::doughnut::DummyDoughnut>>, srml_system::ChainContext<doughnut::tests::Runtime>, srml_balances::Module<doughnut::tests::Runtime>, ()>`
    |
    = note: the method `apply_extrinsic` exists but the following trait bounds were not satisfied:
            `[u8; 64] : std::convert::AsRef<[u8]>`

```

--- a/core/sr-primitives/src/testing.rs
+++ b/core/sr-primitives/src/testing.rs
@@ -354,13 +354,13 @@ pub mod doughnut {
 
 	impl DoughnutApi for DummyDoughnut {
 		type AccountId = TestAccountId;
-		type Signature = [u8; 64];
+		type Signature = Vec<u8>;
 		type Timestamp = ();
 		fn holder(&self) -> Self::AccountId { self.holder.clone() }
 		fn issuer(&self) -> Self::AccountId { self.issuer.clone() }
 		fn expiry(&self) -> Self::Timestamp { () }
 		fn payload(&self) -> Vec<u8> { Default::default() }
-		fn signature(&self) -> Self::Signature { [0u8; 64] }
+		fn signature(&self) -> Self::Signature { Default::default() }
 		fn get_domain(&self, _domain: &str) -> Option<&[u8]> { None }
 	}
 }

